build/Targets/Settings.props

```diff
@@ -106,21 +107,25 @@
 
     <!-- https://github.com/dotnet/roslyn/issues/21272 -->
     <GenerateResourceMSBuildRuntime Condition="'$(TargetFramework)' == 'netcoreapp2.0'">CurrentRuntime</GenerateResourceMSBuildRuntime>
-   
-    <!-- 
+
+    <!--
       Official build:
        - Build standalone Portable PDBs to reduce the size of the binaries.
        - Convert Portable PDBs to Windows PDBs and publish the converted PDBs to Symbol Store to allow WinDBG, 
          Watson and other tools to find symbol format they understand.
-        
+
       Jenkins build:
        - Embed PDBs to make it easier to debug Jenkins crash dumps.
-       
+
       Developer build:
        - Embed PDBs to be consistent with Jenkins builds.
+
+      Cross-platform (developer + Jenkins) build:
+       - Use Portable PDBs.
     -->
     <RoslynDebugType Condition="'$(OfficialBuild)' == 'true'">portable</RoslynDebugType>
     <RoslynDebugType Condition="'$(OfficialBuild)' != 'true'">embedded</RoslynDebugType>
+    <RoslynDebugType Condition="'$(OS)' != 'Windows_NT'">portable</RoslynDebugType>
```

Comment:
Should we revisit this logic now? I think it's possible for the Unix builds to now be completely consistent with our Windows builds here for PDBs. The logic for embedding and portable should work in both places. 

CC @tmat as he's been working on this area.

Reply:
I agree. There is no need for different settings on Windows.

---
In reply to: [150285679](https://github.com/dotnet/roslyn/pull/23113#discussion_r150285679) [](ancestors = 150285679)